sandbox.go

```diff
@@ -700,6 +700,49 @@ func (sb *sandbox) releaseOSSbox() {
 	osSbox.Destroy()
 }
 
+func (sb *sandbox) restoreSandbox(option []SandboxOption) error {
+	var routes []*types.StaticRoute
+
+	// restore sb.config
+	sb.processOptions(option...)
+	sb.restorePath()
+
+	// restore osl sandbox
+	Ifaces := make(map[string][]osl.IfaceOption)
+	ifaceOptions := make([][]osl.IfaceOption, len(sb.endpoints))
+	for j, ep := range sb.endpoints {
+		ep.Lock()
+		joinInfo := ep.joinInfo
+		i := ep.iface
+		ep.Unlock()
+		ifaceOptions[j] = append(ifaceOptions[j], sb.osSbox.InterfaceOptions().Address(i.addr), sb.osSbox.InterfaceOptions().Routes(i.routes))
+		if i.addrv6 != nil && i.addrv6.IP.To16() != nil {
+			ifaceOptions[j] = append(ifaceOptions[j], sb.osSbox.InterfaceOptions().AddressIPv6(i.addrv6))
+		}
+		if i.mac != nil {
+			ifaceOptions[j] = append(ifaceOptions[j], sb.osSbox.InterfaceOptions().MacAddress(i.mac))
+		}
+		Ifaces[fmt.Sprintf("%s+%s", i.srcName, i.dstPrefix)] = ifaceOptions[j]
+		if joinInfo != nil {
+			for _, r := range joinInfo.StaticRoutes {
+				routes = append(routes, r)
+			}
+		}
+		if ep.needResolver() {
+			sb.startResolver()
+		}
+	}
+	gwep := sb.getGatewayEndpoint()
+	joinInfo := gwep.joinInfo
```

Comment:
@aboch  we should check if `gwep` is nil here for restore `host` and `none` network mode

This changes along with the changes in comment https://github.com/docker/libnetwork/pull/1244/files#r66715221
is https://github.com/coolljt0725/libnetwork/commit/ab94339bf57b4203c312589da6d0419f87e2d91d